src/pages/index.js

```diff
@@ -29,6 +30,45 @@ const addCardForm = addCardModal.querySelector(".modal__form");
 
 const profileEditForm = profileEditModal.querySelector(".modal__form");
 
+const api = new Api({
+  baseUrl: "https://around-api.en.tripleten-services.com/v1",
+  headers: {
+    authorization: "fe7e07a4-81c5-490b-807b-e6a7cec619a0",
+    "Content-Type": "application/json",
+  },
+});
+
+let cardSection;
+
+api
+  .getInitialCards()
+  .then((res) => {
+    cardSection = new Section(
+      {
+        items: res,
+        renderer: (data) => {
+          const card = renderCard(data);
+          cardSection.addItem(card);
+        },
+      },
+      selectors.cardSection
+    );
+    cardSection.renderItems();
+  })
+  .catch((err) => {
+    console.error(err);
+  });
+
+api
+  .getUserInfo()
```

Comment:
Better to use `Promise.all` to load all the initial data at once

Reference: https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Promise/all

Reply:
I created the new code using "promise.all."